core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/NetworkV1ToNetworkV4Converter.java

```diff
@@ -29,18 +33,26 @@ public NetworkV4Request convertToNetworkV4Request(Pair<NetworkV1Request, Detaile
             key = new NetworkV1Request();
         }
 
-        NetworkV4Request response = new NetworkV4Request();
+        NetworkV4Request request = new NetworkV4Request();
 
         switch (network.getValue().getCloudPlatform()) {
             case "AWS":
-                response.setAws(getAwsNetworkParameters(Optional.ofNullable(key.getAws()), value));
+                request.setAws(getAwsNetworkParameters(Optional.ofNullable(key.getAws()), value));
                 break;
             case "AZURE":
-                response.setAzure(getAzureNetworkParameters(Optional.ofNullable(key.getAzure()), value));
+                request.setAzure(getAzureNetworkParameters(Optional.ofNullable(key.getAzure()), value));
+                break;
+            case "MOCK":
+                request.setMock(getMockNetworkParameters(Optional.ofNullable(key.getMock()), value));
                 break;
             default:
         }
-        return response;
+        return request;
+    }
+
+    private MockNetworkV4Parameters getMockNetworkParameters(Optional<MockNetworkV1Parameters> mock, EnvironmentNetworkResponse value) {
```

Comment:
The `getIfNottNull()` always get a _not null_ object. So it is useless with `Optional`. 

Please use `Optional` or `getIfNottNull` for easier understanding.